sources/zsh/aliases.d/ligolo-ng

```diff
@@ -0,0 +1 @@
+alias ligolo-proxy="/opt/tools/ligolo-ng/proxy"
```

Comment:
In my opinion, this shouldn't be an alias, but instead a symbolic link placed in `/opt/tools/bin/`